src/qt/pivx/settings/settingsconsolewidget.h

```diff
@@ -44,7 +44,8 @@ class SettingsConsoleWidget : public PWidget
 
 public Q_SLOTS:
     void clear();
-    void message(int category, const QString& message, bool html = false);
+    void message(int category, const QString &msg) { message(category, msg, false); }
+    void message(int category, const QString &message, bool html);
```

Comment:
This message method doesn't seems to be a SLOT anymore (not connected to any signal). Should be private method.

Reply:
going to leave this as-is for now, as `settingsconsolewidget` is mostly mirroring the old `rpcconsole` (which is still used). can revisit after that code cleanup is done.